sqlwhat/sct_context.py

```diff
@@ -0,0 +1,9 @@
+from protowhat.sct_context import create_sct_context, get_checks_dict
+from sqlwhat import checks
+
+SCT_DICT = get_checks_dict(checks)
+SCT_CTX = create_sct_context(SCT_DICT)
+
+# put on module for easy * importing
+globals().update(SCT_CTX)
```

Comment:
Isn't worth it to keep the old comment from sct_syntax.py(If it is still correct?):
'used in test_exercise, so that scts without Ex() don't run immediately'

Reply:
The comment wasn't very clear and no longer completely applicable.
The documentation of `create_sct_context` should make sure to explain what it returns.